.github/workflows/main.yaml

```diff
@@ -147,3 +160,48 @@ jobs:
         if: ${{ ! (startsWith(matrix.platform, 'android') && matrix.config == 'debug') }}
         with:
           targets: ${{ needs.initialize.outputs.targets }}
+          test_artifacts_key: ${{ env.TEST_ARTIFACTS_KEY }}
+
+  test:
+    needs: [initialize, docker-build-image, build]
+    permissions: {}
+    # TODO(b/372303096): Should have dedicated runner?
+    runs-on: [self-hosted, chrobalt-linux-runner]
+    name: ${{ matrix.name }}_tests
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: ${{ fromJson(needs.initialize.outputs.platforms) }}
+        include: ${{ fromJson(needs.initialize.outputs.includes) }}
+        config: [devel]
+    container: ${{ needs.docker-build-image.outputs.docker_tag }}
+    env:
+      TMPDIR: /__w/_temp
+      TEST_RESULTS_DIR: ${{ matrix.name }}_test_results
+      TEST_ARTIFACTS_KEY: ${{ matrix.platform }}_${{ matrix.name }}_test_artifacts
+      IS_POSTSUBMIT: ${{ github.event_name == 'schedule' || github.event_name == 'push' }}
```

Comment:
`IS_POSTSUBMIT` is unused. Please remove.

Reply:
Done.